core/che-core-api-core/src/main/java/org/eclipse/che/api/core/cors/CheCorsFilter.java

```diff
@@ -11,33 +11,21 @@
  */
 package org.eclipse.che.api.core.cors;
 
-import static org.apache.catalina.filters.CorsFilter.DEFAULT_ALLOWED_ORIGINS;
-import static org.apache.catalina.filters.CorsFilter.PARAM_CORS_ALLOWED_HEADERS;
-import static org.apache.catalina.filters.CorsFilter.PARAM_CORS_ALLOWED_METHODS;
-import static org.apache.catalina.filters.CorsFilter.PARAM_CORS_ALLOWED_ORIGINS;
-import static org.apache.catalina.filters.CorsFilter.PARAM_CORS_EXPOSED_HEADERS;
-import static org.apache.catalina.filters.CorsFilter.PARAM_CORS_PREFLIGHT_MAXAGE;
-import static org.apache.catalina.filters.CorsFilter.PARAM_CORS_SUPPORT_CREDENTIALS;
-
 import com.google.inject.Singleton;
 import java.io.IOException;
-import java.util.Enumeration;
-import java.util.HashMap;
-import java.util.Map;
+import javax.inject.Inject;
 import javax.servlet.Filter;
 import javax.servlet.FilterChain;
 import javax.servlet.FilterConfig;
-import javax.servlet.ServletContext;
 import javax.servlet.ServletException;
 import javax.servlet.ServletRequest;
 import javax.servlet.ServletResponse;
 import org.apache.catalina.filters.CorsFilter;
 
 /**
  * The special filter which provides filtering requests in according to settings which are set to
- * {@link CorsFilter}. More information about filter and parameters you can find in documentation.
- * The class contains business logic which allows to get allowed origin from any endpoint as it is
- * used by export workspace.
+ * {@link CorsFilter}. The class allows to configure parameters, such as support credentials in
+ * request, or
```

Comment:
1. Looks like the sentence is not finished.
2. Actually, this class doesn't allow to configure parameters directly, so consider adding a link to CheCorsFilterConfig.
3. Please don't be shy and feel free to add yourself to authors.

Reply:
fixed. ty